### PR TITLE
red/master

### PR DESCRIPTION
**Pull request by RSKYS**
`audio/tauonmb`
- Pillow version 9.1.0+ required, resampling method is not supported by older versions;
Results, you won't get album artworks.
- Please do not modify PHAZOR compiler;
Depends on each version there might be something different.
- There are some new optional dependencies, but I don't have time for them:
**jxlpy**, **pychromecast**
Anybody feel free to add them; I'm gonna mention those in README then.

`development/mdbook`,`system/nix`

I'm sorry but I'm not really following changes other than my own attempts.
I realized some SBo insiders made changes on my builds, again.
When I took another look they weren't bad; I reverted their changes in last two commits alongside updates.

### CODE REVIEW

**Comment by willysr** [comment]
can you make a single commit on nix and mdbook?
you can try to rebase and pick squash

**Comment by RSKYS** [comment]
Rebased.

**Comment by willysr** [comment]
@aclemons did you test this?

File: development/mdbook/mdbook.SlackBuild [hunk 1]
```diff
@@ -29,7 +29,7 @@
 cd $(dirname $0) ; CWD=$(pwd)
 
 PRGNAM=mdbook
-VERSION=${VERSION:-0.4.15}
+VERSION=${VERSION:-0.4.18}
 SRCNAM=mdBook
 BUILD=${BUILD:-2}
```
Comment (willysr) [inline development/mdbook/mdbook.SlackBuild:34]: When you bump VERSION, you should reset BUILD to 1

**Review by willysr — COMMENTED** [review]
BUILD should be reset to 1 when you change VERSION

**Comment by willysr** [comment]
next time, make single PR for every commit, so we can take individual PR. Otherwise we would have to wait until everything is checked out

**Comment by RSKYS** [comment]
I took brief look
cargo.lock on mdbook project still using same boomer versions and nothing needs updates;
Phew, lovely SlackBuilds moment!

I push the correct build version and everything should basically work smooth I guess.

**Comment by RSKYS** [comment]
Hey.. I was exploring around and realized tekore has newer version
But I'd probably wait for this to stage on someone's branch, merge or whatever
I'd probably push anything new few weeks later

**Comment by aclemons** [comment]
> @aclemons did you test this?

I have not touched this PR yet

**Comment by RSKYS** [comment]
> was exploring around and realized tekore has newer version
But I'd probably wait for these to stage on someone's branch, merge or whatever

There're not very major differences anyway
Works smooth

**Comment by RSKYS** [comment]
> I was exploring around and realized tekore has newer version
But I'd probably wait for these to stage on someone's branch, merge or whatever

There's no major difference anyway
Works smooth

**Comment by willysr** [comment]
merged

**Pull request closed without merging**